js/ui/audio_worker.ts

```diff
@@ -0,0 +1,47 @@
+
```

Comment:
Don't know if you care about adding the copyrights and license and whatnot. I know I've forgotten.